Tools/Scripts/webkitpy/w3c/common.py

```diff
@@ -58,13 +58,12 @@ def read_credentials(host, credentials_json):
 
 
 class WPTPaths:
-    CHECKOUT_DIRECTORY = ["w3c-tests"]
-    WPT_CHECKOUT_PATH = CHECKOUT_DIRECTORY + ["web-platform-tests"]
+    WPT_CHECKOUT_PATH = "web-platform-tests"
 
     @staticmethod
     def checkout_directory(finder):
-        return finder.path_from_webkit_outputdir(*WPTPaths.CHECKOUT_DIRECTORY)
+        return os.path.join(os.path.dirname(finder.webkit_base()))
```

Comment:
Based on my comments re: `wpt_checkout_path`, I think we should just flip which way around they call each other and make this call `wpt_checkout_path` and then find the `dirname` of that.